text/000-build-notifications.md

```diff
@@ -0,0 +1,71 @@
+- Start Date: 2018-04-27
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Include `ember-cli-build-notifications` by default
+
+## Summary
+
+Adding ember-cli-build-notifications as a default addon will increase developer productivity and improve the feedback cycle of day-to-day development. It will also assist new ember devs by quickly notifying them of errors they do not yet know to look at for. The default settings should should be verbose, as that would be most helpful to new Ember developers. 
+
+## Motivation
+
+Confusion as a result of the state of the build consumes a surprising amount of developer cycles. For new developers, this can be especially frustrating: 
+
+> There’s no ide integration that shows you errors without compilation. The most frustrating thing about this is that I would write broken handlebars, and my server would refresh, but nothing would change. Eventually, I would see the error in the terminal where I was serving Ember and fix it. [My first 6 months using Ember: A retrospective](https://medium.com/@Realrobwebb/my-first-6-months-using-ember-a-retrospective-a5ecf3259b09)
+
+There are several parallel paths that we can take to communicate the state of a build. Recent developments in the new error page are one of them.  Having `ember-cli-build-notifications` alert on **all** builds is another. Not all problems result in a build error, so the absence of a notification is also a useful tool in debugging.
+
+Some of these scenarios include: 
+* Addons, like `ember-cli-typescript` or tools like `watchman` breaking and not to starting a new build
+* Saves in linked packages not causing builds to start
+* Page refreshed not happening after successful builds due to livereload disconnecting
+* Making only style changes and not realizing the new css was already pushed
+
+Build notifications give automatic insight into these scenarios in an unobtrusive way. 
+
+## Detailed design
+1. Include `ember-cli-build-notifications` as part of the default kit
```

Comment:
For people like me, who didn't know of this addon before, it would be helpful to have a short summary what it actually does, so we don't have to look for the github page. 

Also maybe elaborate how it would solve the scenarios you described in the previous section?